.travis-ci.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+if [[ $TASK = 'nosetests' ]]; then
+    pip install json-spec
+    nosetests
+elif [[ $TASK = 'karma' ]]; then
+    export DISPLAY=:99.0
+    sh -e /etc/init.d/xvfb start
+    npm install -g grunt-cli
```

Comment:
I'm not sure this will work on travis. Does npm have global permissions? I guess we'll find out.